examples/pouchPersisted/index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <script src="../../src/lib/infusion/infusion-all.js"></script>
+        <script src="../../src/lib/pouchdb/dist/pouchdb.js"></script>
+        <script src="../../src/js/pouchPersisted.js"></script>
+        <link rel="stylesheet" type="text/css" href="../../src/lib/infusion/src/components/inlineEdit/css/InlineEdit.css" />
+        <link rel="stylesheet" type="text/css" href="../../src/lib/infusion/src/framework/core/css/fluid.css" />
+    </head>
+
+    <body>
+        <h1>pouchPersisted Grade Example</h1>
+        <p>
+            This is a simple example of an inlineEdit component that persists its value to PouchDB.
+        </p>
+        <p>
+            Instructions:
+            <ul>
+                <li>
+                    Edit the text.
+                </li>
+                <li>
+                    Reload the page; you'll see your text has persisted.
+                </li>
+                <li>
+                    The log will also update.
+                </li>
+            </ul>
+        </p>
+        <h2>Edit Here</h2>
+        <div class="floec-persistedInlineEdit">
+            <div class="flc-inlineEdit-text">
+
+            </div>
+            <h3>Persistence Log</h3>
+            <div class="floec-persistedInlineEdit-loggingArea">
+
+            </div>
+        </div>
+    <script>
+        fluid.defaults("floe.dashboard.pouchPersistedInlineEdit", {
+            gradeNames: ["floe.dashboard.pouchPersisted", "fluid.inlineEdit"],
+            invokers: {
+                // Override the standard timestamp-based approach to a
+                // fixed ID, for the sake of this example
+                "setPouchId": {
```

Comment:
It's the easiest approach for demonstrating a simple example of grade persistence that didn't require creating additional grades to do (FE) searching and filtering. Some of that functionality exists in grades from the prototype work, but they need refactoring and tests. I'll add a comment to that effect.